pkg/local_object_storage/metabase/metadata.go

```diff
@@ -191,9 +202,25 @@ type SearchCursor struct {
 	ValIDOff int
 }
 
+// splits VAL_DELIM_OID.
+func splitValOID(b []byte) ([]byte, []byte, error) {
+	if len(b) < oid.Size {
+		return nil, nil, fmt.Errorf("too short len %d", len(b))
+	}
+	idOff := len(b) - oid.Size
+	val, ok := bytes.CutSuffix(b[:idOff], attributeDelimiter)
```

Comment:
It's just a check for attributeDelimiter at the known offset and subslice.

Reply:
done